cmd/testpeers.go

```diff
@@ -38,19 +72,30 @@ func newTestPeersCmd(runFunc func(context.Context, io.Writer, testPeersConfig) e
 
 	bindTestFlags(cmd, &config.testConfig)
 	bindTestPeersFlags(cmd, &config)
+	bindP2PFlags(cmd, &config.P2P)
+	bindDataDirFlag(cmd.Flags(), &config.DataDir)
+	bindTestLogFlags(cmd.Flags(), &config.Log)
 
 	return cmd
 }
 
 func bindTestPeersFlags(cmd *cobra.Command, config *testPeersConfig) {
 	const enrs = "enrs"
-	cmd.Flags().StringSliceVar(&config.ENRs, "enrs", nil, "[REQUIRED] Comma-separated list of each peer ENR address.")
-	cmd.Flags().StringSliceVar(&config.P2PRelays, "p2p-relays", []string{"https://0.relay.obol.tech,https://2.relay.obol.tech"}, "Comma-separated list of each peer P2P relay address.")
+	cmd.Flags().StringSliceVar(&config.ENRs, enrs, nil, "[REQUIRED] Comma-separated list of each peer ENR address.")
+	cmd.Flags().DurationVar(&config.KeepAlive, "keep-alive", 30*time.Minute, "Time to keep TCP node alive after test completion, so connection is open for other peers to test on their end.")
+	cmd.Flags().DurationVar(&config.LoadTestDuration, "load-test-duration", 30*time.Second, "Time to keep running the load tests in seconds. For each second a new continuous ping instance is spawned.")
 	mustMarkFlagRequired(cmd, enrs)
 }
 
-func supportedPeerTestCases() map[testCaseName]func(context.Context, *testPeersConfig, string) testResult {
-	return map[testCaseName]func(context.Context, *testPeersConfig, string) testResult{
+func bindTestLogFlags(flags *pflag.FlagSet, config *log.Config) {
+	flags.StringVar(&config.Format, "log-format", "console", "Log format; console, logfmt or json")
+	flags.StringVar(&config.Level, "log-level", "info", "Log level; debug, info, warn or error")
+	flags.StringVar(&config.Color, "log-color", "auto", "Log color; auto, force, disable.")
+	flags.StringVar(&config.LogOutputPath, "log-output-path", "", "Path in which to write on-disk logs.")
+}
+
+func supportedPeerTestCases() map[testCaseName]func(context.Context, *testPeersConfig, host.Host, p2p.Peer) testResult {
+	return map[testCaseName]func(context.Context, *testPeersConfig, host.Host, p2p.Peer) testResult{
```

Comment:
Nit: maybe have a `testCase` type for `func(context.Context, *testPeersConfig, host.Host, p2p.Peer) testResult`? To increase readability.